charts/lakefs/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: lakefs
 description: A Helm chart for running LakeFS on Kubernetes
 type: application
-version: 0.8.11
-appVersion: 0.94.1
+version: 0.8.12
```

```suggestion
version: 0.9.0
```
This release adds a feature to the chart.

---

100%